packages/sandbox/src/app/sandbox/index.ts

```diff
@@ -63,4 +64,8 @@ export const sandboxItems: SandBoxSectionType[] = [
     title: 'Tags',
     items: [{ title: 'Tag', SandBox: TagSandbox }],
   },
+  {
+    title: 'Boolean Inputs',
```

Comment:
WDYT to rename `Inputs` to `Forms` and also add checkbox in it?

Reply:
We will also have the switch component in it. So I created another section to avoid having a large one. But if you prefer we can merge both sure